utils/das_data_sorter.py

```diff
@@ -0,0 +1,75 @@
+import pandas as pd 
+import argparse 
+from math import ceil
+
+# accepts terminal commands
+parser = argparse.ArgumentParser()
+parser.add_argument("--file", help="Input CSV file", action="store")
+parser.add_argument("--output", help="Returns the filtered data", default="filtered_data.csv", action="store")
+parser.add_argument("--unit", help="Specify time units (seconds, s, or minutes, m)", default="seconds", action="store")
+args = parser.parse_args()
+
+# initialising data
+data = pd.read_csv(args.file)
+milliseconds = data["time"]
+gps_course = data["gps_course"]
+gps_speed = data["gps_speed"]
+tempC = data["thermoC"]
+tempF = data["thermoF"]
+cadence = data["cadence"]
+power = data["power"]
+reed_vel = data["reed_velocity"]
+reed_dis = data["reed_distance"]
+
+time = []
+if args.unit == "seconds" or args.unit == "s":
+    time = milliseconds/1000
```

Comment:
It doesn't make sense to overwrite the time variable with an integer if you declare it as an empty array above. Maybe you should rename the variable and declare it as `None`?

Reply:
Ah, the milliseconds variable is an array of data, not an integer. So at the end of this code block, time variable becomes an array of data (divided by 1000) as well.

Reply:
ohhhh, yup. I misread that thank you. data is in a dataframe.